en/concept/mixing.md

```diff
@@ -74,16 +76,16 @@ For a multicopter things are a bit different: control 0 (roll) is connected to a
 * 1: RC pitch
 * 2: RC yaw
 * 3: RC throttle
-* 4: RC mode switch
-* 5: RC aux1
-* 6: RC aux2
-* 7: RC aux3
+* 4: RC mode switch (Passthrough of RC channel mapped by [RC_MAP_FLAPS](../advanced/parameter_reference.md#RC_MAP_FLAPS))
+* 5: RC aux1 (Passthrough of RC channel mapped by [RC_MAP_AUX1](../advanced/parameter_reference.md#RC_MAP_AUX1))
+* 6: RC aux2 (Passthrough of RC channel mapped by [RC_MAP_AUX2](../advanced/parameter_reference.md#RC_MAP_AUX2))
+* 7: RC aux3 (Passthrough of RC channel mapped by [RC_MAP_AUX3](../advanced/parameter_reference.md#RC_MAP_AUX3))
 
 > **Note** This group is only used to define mapping of RC inputs to specific outputs during *normal operation* (see [quad_x.main.mix](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/mixers/quad_x.main.mix#L7) for an example of AUX2 being scaled in a mixer).
   In the event of manual IO failsafe override (if the PX4FMU stops communicating with the PX4IO board) only the mapping/mixing defined by control group 0 inputs for roll, pitch, yaw and throttle are used (other mappings are ignored).
 
 
-### Control Group #6 (First Payload)
+### Control Group #6 (First Payload) {#control_group_6}
 
 * 0: function 0 (default: parachute)
```

Comment:
@bkueng One big question that is not answered is what values can you use?

My assumption would be that the payload group is for whatever custom stuff you want. If we had a defined payload group we wouldn't want people to mess with it and overload its use. But then I see default is marked as "parachute" so I am wondering why this isn't called "parachute" to reserve it?

Can you clarify?

Reply:
Yes parachute is simply incorrect, and should be removed. As you also mention, only one module is allowed to publish a certain control_group topic at a given time, otherwise you will run into problems.